tz/digest.go

```diff
@@ -22,6 +24,8 @@ type digest struct {
 }
 
 // New returns a new [hash.Hash] computing the Tillich-Zémor checksum.
+// The Hash also implements encoding.BinaryMarshaler and encoding.BinaryUnmarshaler
```

Comment:
`[]` for documentation links (you can import `encoding` and check for compliance, this would make links work as well.).

Reply:
Updated